### PR TITLE
NFC: Treat group switches like normal switches

### LINKED ISSUE #376: NFC - Group as a Switch - Empty List
If I am using normal switches for programming a NFC-Tag I can select between ON, OFF and TOGGLE.

When I am using a Group defined as Switch I just get an empty list when long-pressing it.

### PR DESCRIPTION
**Pull request by mueller-ma**
Fixes #376 

Signed-off-by: mueller-ma <mueller-ma@users.noreply.github.com>

### CODE REVIEW

**Comment by mueller-ma** [comment]
I dont have a nfc device, so I only see that the action selection popup comes up.

File: mobile/src/main/java/org/openhab/habdroid/model/OpenHABNFCActionList.java [hunk 1]
```diff
@@ -16,64 +16,71 @@
...
-		return this.actionNames.toArray(new String[this.actionNames.size()]);
-	}
-	
-	public String[] getCommands() {
-		return this.actionCommands.toArray(new String[this.actionCommands.size()]);
-	}
+    /**
+     * Creates two list with all available nfc actions for an item, one with a user visble label
+     * (name) and one with the command
+     * @param openHABWidget
```
[the remark this answers is not in the document]
Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/model/OpenHABNFCActionList.java]: I think thats very self explanatory. Thoughts?

File: mobile/src/main/java/org/openhab/habdroid/model/OpenHABNFCActionList.java [hunk 2]
```diff
@@ -16,64 +16,71 @@
...
+     * @param ctx
+     */
+    public OpenHABNFCActionList(OpenHABWidget openHABWidget, Context ctx) {
```
[the remark this answers is not in the document]
Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/model/OpenHABNFCActionList.java]: We should make an exception for "openHAB" or start removing it from all class and variable names.

[the remark this answers is not in the document]
Comment (maniac103) [reply mobile/src/main/java/org/openhab/habdroid/model/OpenHABNFCActionList.java]: Or both ;-) It's redundant in most cases indeed.

[the remark this answers is not in the document]
Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/model/OpenHABNFCActionList.java]: Should we do the whole removal in one PR or everytime someone changes a file/method like in this PR?

[the remark this answers is not in the document]
Comment (maniac103) [reply mobile/src/main/java/org/openhab/habdroid/model/OpenHABNFCActionList.java]: I'd probably do it in a separate PR (or multiple separate PRs), as class renames tend to be invasive. E.g. renaming OpenHABWidget or OpenHABItem will touch a lot of code.

**Pull request merged**